### DISCUSSION TITLE
Discord Server Now Available!

### DISCUSSION THREAD

**Discussion by synman** [synman/Ender-3-S1-Pro-Firmware#43] [Features]
There's a discord server now for real-time discussion of the firmware.  I left the server description pretty vague to open up the possibility for broader discussions.

https://discord.gg/bj7na3PV

**Comment by shardsofaperture** [comment]
Link is currently expired, any chance to repost?

**Reply by synman** [reply]
https://discord.gg/tEhMB3tz

**Comment by m3shary** [comment]
both links are expired, could you repost?
